src/manager.rs

```diff
@@ -44,6 +46,24 @@ lazy_static! {
     };
 }
 
+// Workaround the UNC path on Windows, see https://github.com/rust-lang/rust/issues/42869.
```

Comment:
Ah, https://github.com/rust-lang/rust/issues/42869, I [know it well](https://github.com/alexcrichton/cargo-vendor/pull/71#issuecomment-387570072) (unfortunately).

Reply:
Haha, so you've also been there before. Such a small world! :)